src/python/sdk/examples/event.py

```diff
@@ -1,88 +1,54 @@
-from os import environ
-
-import httpx
-import rel
-import websocket
-import jwt
-
-JWT_SECRET_DEFAULT = "econia_0000000000000000000000000"
-WS_URL_LOCAL_DEFAULT = "ws://0.0.0.0:3001"
-WS_CHANNEL_DEFAULT = "fill_event"
-
-
-def on_message(ws, message):
-    print(message)
-
-
-def on_error(ws, error):
-    print(error)
-
-
-def on_close(ws, close_status_code, close_msg):
-    print("### closed ###")
-
-
-def on_open(ws):
-    print("Opened connection")
-
-
-def get_ws_host() -> str:
-    url = environ.get("WS_URL")
-    if url == None:
-        url_in = input(
-            "Enter the URL of the WebSocket host (enter nothing to default to local OR re-run with WS_URL environment variable)\n"
-        ).strip()
-        if url_in == "":
-            return WS_URL_LOCAL_DEFAULT
-        else:
-            return url_in
+import paho.mqtt.client as mqtt
+import json
+
+# This is an example on how to use subscribe to events emitted by the DSS.
+
+# Once you have a DSS running (see
+# https://econia.dev/off-chain/dss/data-service-stack on how to do that), you
+# can run this script by running:
+#
+# $ poetry install
+# $ poetry run event
+#
+# It will subscribe to all fill events and to place limit order events on
+# market 7.
+#
+# Each MQTT message has a topic and a payload. You can subscribe to topics.
+# When specifying the topic you want to subscribe to, you can put a + instead
+# of a value to subscribe to all events matching the rest of the topic.
+#
+# Example:
+#
+# We use the topic fill/MARKET_ID/USER_ADDRESS/CUSTODIAN_ID. Fill events by the
+# user 0xdeadbeef with custodian ID 1 on market 3 will have fill/3/0xdeadbeef/1
+# as a topic. But you could subscribe to fill/+/0xdeadbeef/+ to get all fill
+# events from the user 0xdeadbeef.
+
+def on_connect(client, userdata, flags, reason_code, properties):
```

Comment:
Per eng call note that this should be generalizable for use with local testing, Python trade script, etc.

- [ ] See old event.py format for generalized approach